Source/ui/EditToneTab.cpp

```diff
@@ -93,8 +1360,276 @@ void EditToneTab::buttonStateChanged (juce::Button* button)
 
 void EditToneTab::comboBoxChanged (juce::ComboBox* button)
 {
-    if (button == &toneSelectionComboBox)
-    {
-        updateValues();
-    }
+    sendSysexPatchToneChange();
+}
+
+void EditToneTab::sendSysexPatchToneChange1(uint32_t address, uint8_t value)
+{
+      uint8_t data[5];
+      data[0] = 0x00; // address MSB
+      data[1] = 0x08; // address
+      data[2] = 0x28 + toneCount;  // address
+      data[3] = address & 127;  // address LSB
+      data[4] = value;                 // data
+      uint32_t checksum = 0;
+      for (size_t i = 0; i < 5; i++) {

```

Comment:
nit: this checksum computation could be a shared function instead of a copy paste?